src/components/Header.tsx

```diff
@@ -42,7 +41,7 @@ const Header = (props: BoxProps) => {
                   <Image
                     priority
                     src={mascotIcon}
-                    alt="Go to home page"
+                    alt=""
```

Comment:
I have removed this since it was recommended by Sa11y utility:
<img width="498" alt="CleanShot 2022-01-05 at 19 31 36@2x" src="https://user-images.githubusercontent.com/2677072/148269949-0cf9a2d4-bb20-4ac1-93b8-8bbbd79ecda8.png">